exporter/containerimage/writer.go

```diff
@@ -277,7 +277,7 @@ func (ic *ImageWriter) commitDistributionManifest(ctx context.Context, ref cache
 
 	for i, desc := range remote.Descriptors {
 		// oci supports annotations but don't export internal annotations
-		if oci {
+		if oci || compressionType == compression.NydusBlob {
```

Comment:
Maybe `oci-mediatypes` should be always enabled if `compression=nydus` in `*imageExporter.Resolve`?